lib/handlers/buildModels.js

```diff
@@ -3,12 +3,117 @@
 // external modules
 var _           = require('lodash');
 var vasync      = require('vasync');
+var assert      = require('assert-plus');
 
 // internal files
 var reqHelpers  = require('../reqHelpers');
 var logHelpers  = require('../logHelpers');
 var errorTypes  = require('../errors');
 
+/**
+ * Internal method for fetching any models that are resolved from
+ * the model bucket.
+ *
+ * @private
+ * @param {Object} req request object
+ * @param {Object} res response object
+ * @param {Object} model model object
+ * @param {Function} callback callback when fetch is done
+ * @returns {undefined}
```

Comment:
Nitpick, @function getModel.

Maybe this repo doesn't have JSDoc linting enforced.